README.md

```diff
@@ -50,3 +48,9 @@ may not work properly on specific/all platforms.
 - ATMEL AVR Series (incl. Arduino): avr-gcc
 - x86, x86_64: gcc, clang
 
+Contributors
+------------
+
+- eflukx
+- hasegaw
+- ryohack
```

Comment:
My apologies. The typo is fixed in commit b39c3a5b2ee3a0efca42689f769414fb09fc0008.

Takeshi